scripts/tfenv-install-terraform.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+
+if [ -f "~/.tfenv" ]; then
+  rm -rf ~/.tfenv
+fi
+
+git clone https://github.com/tfutils/tfenv.git ~/.tfenv
+
+if [ ! -f "/usr/bin/sudo" ]; then
+  apt install -y sudo
+fi
+
+if [ -f "/usr/local/bin/terraform" ]; then
+  sudo rm /usr/local/bin/terraform
+fi
+
+sudo ln -s ~/.tfenv/bin/* /usr/local/bin
```

Comment:
is this all needed? tfenv should be handling it for you? and can't you just invoke tfenv directly at the path?

Reply:
This was to workaround an issue with missing sudo on the vsts-aks-agent.  Will look into installing tfenv in the home directory instead.

Reply:
tfenv installed in home directory.